deploy/cloudformation/static-host.yml

```diff
@@ -115,20 +107,20 @@ Resources:
         Enabled: true
         HttpVersion: http2
         PriceClass: PriceClass_100
-        ViewerCertificate: !If
-          - NoSSL
-          - !Ref AWS::NoValue
-          - AcmCertificateArn: !Ref AcmCertificateArn
-            MinimumProtocolVersion: TLSv1.1_2016
-            SslSupportMethod: sni-only
-        Comment: !If
-          - NoFQDN
-          - !Ref AWS::NoValue
-          - !Ref FQDN
-        Aliases: !If
-          - NoFQDN
-          - !Ref AWS::NoValue
-          - - !Ref FQDN
+        ViewerCertificate: 
+          AcmCertificateArn: 
+              Fn::ImportValue: !Join [':', [!Ref InfrastructureStackName, 'ACMCertificateARN']]
+          MinimumProtocolVersion: TLSv1.1_2016
+          SslSupportMethod: sni-only
+        Comment: !Sub
```

Comment:
Should L115-118 match L119-123 in format and indentation? It seems like one or the other might not produce expected behavior here. @hanstra , can you point to where this is deployed to a CloudFront so we can take a look?

Reply:
I tore this down earlier in the week but can rebuild if desired. I recall Alias did not work when pulled up a line, but comment did. I did not review closely enough to know what the actual resulting Comment looked like.

Reply:
Alias is an array; comment is a string. So the syntax is correct.